tests/common/test_event.py

```diff
@@ -269,13 +272,29 @@ def test_periodic_emits_after_elapsed_delta(self, mock_event):
     @patch('azurelinuxagent.common.event.EventLogger.add_event')
     def test_periodic_forwards_args(self, mock_event):
         event.__event_logger__.reset_periodic()
+        event_time = datetime.utcnow().__str__()
+        event.add_periodic(logger.EVERY_DAY, "FauxEvent", op=WALAEventOperation.Log, is_success=True, duration=0,
+                           version=str(CURRENT_VERSION), message="FauxEventMessage", evt_type="", is_internal=False,
+                           log_event=True, force=False)
+        mock_event.assert_called_once_with("FauxEvent", op=WALAEventOperation.Log, is_success=True, duration=0,
+                                           version=str(CURRENT_VERSION), message="FauxEventMessage", evt_type="",
+                                           is_internal=False, log_event=True)
+
+    @patch("azurelinuxagent.common.event.datetime")
+    @patch('azurelinuxagent.common.event.EventLogger.add_event')
+    def test_periodic_forwards_args_default_values(self, mock_event, mock_datetime):
+        event.__event_logger__.reset_periodic()
+        event.add_periodic(logger.EVERY_DAY, "FauxEvent", message="FauxEventMessage")
+        mock_event.assert_called_once_with("FauxEvent", op=WALAEventOperation.Unknown, is_success=True, duration=0,
+                                           version=str(CURRENT_VERSION), message="FauxEventMessage", evt_type="",
+                                           is_internal=False, log_event=True)
 
-        event.add_periodic(logger.EVERY_DAY, "FauxEvent")
-        mock_event.assert_called_once_with(
-            "FauxEvent",
-            duration=0, evt_type='', is_internal=False, is_success=True,
-            log_event=True, message='', op=WALAEventOperation.Unknown,
-            version=str(CURRENT_VERSION))
+    @patch("azurelinuxagent.common.event.EventLogger.add_event")
+    def test_add_event_default_variables(self, mock_add_event):
+        add_event('test', message='test event')
+        mock_add_event.assert_called_once_with('test', duration=0, evt_type='', is_internal=False, is_success=True,
+                                               log_event=True, message='test event', op='Unknown',
```

Comment:
Why `Unknown` instead of `WALAEventOperation.Unknown`?

Reply:
It was just for testing, wanted to see if the passed values were coming.

Reply:
but why the change?

Reply:
Oh. Now I understood what you both meant. Yes, I'll revert it to `WALAEventOperation.Unknown`.